workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -692,6 +692,19 @@ pre[class*='language-'] {
 	}
 }
 
+.search-banner {
+	&::after {
+		top: 155px !important;
```

Comment:
in general we try to avoid !important unless we really need it

Reply:
in this case, the home banner is different from the search banner, so I had to add this new class to make the necessary adjustments. Once I'm using learn-banner class + search-banner class, I need to set the properties that need to subscribe.